velox/substrait/SubstraitToVeloxExpr.cpp

```diff
@@ -15,8 +15,10 @@
  */
 
 #include "velox/substrait/SubstraitToVeloxExpr.h"
+#include "velox/parse/VariantToVector.h"
```

Comment:
The code in velox/parse is test-only. Let's move this utility into velox/vector (in a separate PR).

Reply:
Hi @mbasmanova, I make a separate PR here to fix this comments. https://github.com/facebookincubator/velox/pull/2679